README.md

```diff
@@ -1 +1,32 @@
-# Sport
\ No newline at end of file
+# Football_teams ⚽
+
+### File structure
+
+- Public folder:
+  - index.html
+  - style.css
+  -script.js
+- Src folder:
+  - server.js
+  - router.js
+  - handler.js
+  - database folder:
+    - db_build.js
+    - db_build.sql
+    - db_connection.js
+  - queries folder:
+    - getData.js
+    - postUser.js
+- config.envPublic folder
```

Comment:
There's an error here, please delete public folder on this last line

Reply:
@jema28 we fix it